test/framework/src/main/java/org/elasticsearch/index/mapper/FieldTypeTestCase.java

```diff
@@ -51,26 +48,6 @@
             copy.setBoost(t.boost() + 1);
             return (T) copy;
         },
-        t -> {
```

Comment:
Should we still test these on string fields?

Reply:
actually yes, these should be moved to the relevent FieldMapper tests

Reply:
ah, they're already tested in FieldMapperTestCase.